pkgs/tools/security/sequoia/default.nix

```diff
@@ -87,5 +87,6 @@ rustPlatform.buildRustPackage rec {
     license = licenses.gpl3;
     maintainers = with maintainers; [ minijackson doronbehar ];
     platforms = platforms.all;
+    broken = true;
```

Comment:
this maintainer is fairly active, I'll open an issue for you to link here so we can direct it to the maintainer.
(marking it as broken is fine)